selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/CheckRestoringWorkspaceAfterStoppingWsAgentProcess.java

```diff
@@ -71,7 +68,6 @@ public void checkRestoreWsAgentByApi() throws Exception {
 
     notificationsPopupPanel.waitWorkspaceAgentIsNotRunning();
```

Comment:
I would check that workspace has status "STOPPED" after `notificationsPopupPanel.waitWorkspaceAgentIsNotRunning();`

Reply:
Actually workspace has status "RUNNING" even when ws-agent is stopped.